fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/RdfStreamProvider.java

```diff
@@ -52,6 +52,11 @@
 
     private static final Logger LOGGER = getLogger(RdfStreamProvider.class);
 
+    private static boolean isTextHtml(final MediaType mediaType) {
```

Comment:
There are two instances of `isTextHtml`. One in:
StreamingBaseHtmlProvider : https://github.com/fcrepo4/fcrepo4/pull/1112/files#diff-479f0eb6453708a90e8c7cc92f0657a6R238

and another, here, in:
RdfStreamProvider.

The one in RdfStreamProvider is not used.